tests/perf-test/ReportScreen.perf-test.js

```diff
@@ -101,6 +101,34 @@ afterEach(() => {
     PusherHelper.teardown();
 });
 
+/**
+ * This is a helper function to create a mock for the addListener function of the react-navigation library.
+ * The reason we need this is because we need to trigger the transitionEnd event in our tests to simulate
+ * the transitionEnd event that is triggered when the screen transition animation is completed.
+ *
+ * P.S: This can't be moved to a utils file because Jest wants any external function to stay in the scope.
+ *
+ * @returns {Object} An object with two functions: triggerTransitionEnd and addListener
+ */
+const createAddListenerMock = () => {
```

Comment:
Maybe it's worth adding this function to the common utils, as we will probably use it in many places in the app?

Reply:
Sadly no. I stated the reasoning already that Jest requires any external functions to be available in the same scope. I already tried creating separate Utils file for it and it didn't work 🤷